data/json/furniture_and_terrain/terrain-doors.json

```diff
@@ -1803,7 +1803,7 @@
     "type": "terrain",
     "id": "t_door_boarded_damaged",
     "name": "boarded up damaged door",
-    "description": "A boarded up wooden door, except some of the planks are fractured or coming loose.  Without repairs, it could be broken down easily.",
+    "description": "A boarded up wooden door, except some planks are fractured or coming loose.  Without repairs, it could be broken down easily.",
```

Comment:
`except for some planks that are`?

Reply:
`except that some planks are`, probably.